src/profile.rs

```diff
@@ -48,9 +48,9 @@ impl From<ProfileTag> for Option<Profile> {
 
 impl From<&Profile> for ProfileTag {
     fn from(profile: &Profile) -> Self {
-        if profile == &Profile::Default {
```

Comment:
I personally think this would look better as a match, but that's not my decision I'd say

Reply:
We can't  do a traditional`match`, unfortunately, as `Profile` is not an `enum`. However, we could do something like:

```rust
match profile {
    p if p == Profile::Default => ProfileTag::Default,
    ...
}
```

I think that'd be great.